app/components/utilities/AAAListButton.tsx

```
@@ -0,0 +1,121 @@
+        'avgTSMPrice' in item
+          ? (item as any).avgTSMPrice
+          : (item as any).historicPrice
+
+      // Skip if below minimum values
+      if (marketValue < minMarketValue || price < minPriceValue) return acc
+
+      const discountedPrice =
+        Math.round(price * (discountPercent / 100) * 100) / 100
+
+      return {
+        ...acc,
```

Biome (1.9.4)

[error] Avoid the use of spread (`...`) syntax on accumulators.

Spread syntax should be avoided on accumulators (like those in `.reduce`) because it causes a time complexity of `O(n^2)`.
Consider methods such as .splice or .push instead.

(lint/performance/noAccumulatingSpread)